docs/modules/ROOT/pages/references/security.adoc

```diff
@@ -0,0 +1,120 @@
+= Security
+
+== Access Management
+
+=== Long Lived Tokens
+
+The default OpenShift 10 year emergency admin token is disabled and replaced by a short lived token.
+
+See xref:oc4:ROOT:references/architecture/emergency_credentials.adoc[].
+
+=== Two Factor Authentication
+
+VSHN managed OpenShift clusters are configured to use two factor authentication through the VSHN SSO for the web console by default.
+The configuration is centrally managed according to best practices.
```

Comment:
which configuration is manage to best practice? the VSHN Login page?

Reply:
Specified OpenID client configuration